content/blog/ultimate-windows-development-environment-guide/index.md

```diff
@@ -603,6 +614,26 @@ Then you're able to navigate to the folder you're looking for, right click, and
 
 There are a myriad of options to choose from and should handle any type of symlink you'd need.
 
+# Additional Configuration {#additional-configuration}
+
+Not quite customization and not quite a "missing feature," there are some things you can do to configure your install to make life better as a developer.
+
+## Long Path Support
+
+Developers with many sub-paths (or those that use package managers like `npm`) can tell you how long their deepest path can get. This can cause problems with Windows, as a limitation exists by default for machines that limits users to paths with 260 characters or less. This can cause havoc and errors when doing automated tasks that exceed this limit. To fix this, you can either use a Registry Editor or modify an existing Group Policy (depending on which edition of Windows you have). The instructions are a bit complex for users not familiar with registry editing, [but HowToGeek provides a great resource](https://www.howtogeek.com/266621/how-to-make-windows-10-accept-file-paths-over-260-characters/) for making this process relatively trivial.
```

Comment:
```suggestion
Developers with many sub-paths (or those that use package managers like `npm`) can tell you just how long their deepest file paths can get. This can cause problems with Windows, because there is a default limit of 260 characters for file paths. This can cause havoc and errors when doing automated tasks that exceed this limit. To fix this, you can either use a Registry Editor or modify an existing Group Policy (depending on which edition of Windows you have). The instructions are a bit complex for users not familiar with registry editing, [but HowToGeek provides a great resource](https://www.howtogeek.com/266621/how-to-make-windows-10-accept-file-paths-over-260-characters/) for making this process relatively trivial.
```